src/openforms/prefill/contrib/stufbg/plugin.py

```diff
@@ -69,8 +70,8 @@ def get_prefill_values(
 
         response_dict = {}
         for attribute in attributes:
-            response_dict[attribute] = glom(
-                data, ATTRIBUTES_TO_STUF_BG_MAPPING[attribute], default=None
-            )
+            value = glom(data, ATTRIBUTES_TO_STUF_BG_MAPPING[attribute], default=None)
+            if "@noValue" not in value:
```

Comment:
This change is unrelated to the ticket but is another bug I found so I wanted to fix it while I was looking at this code.  If a request attribute does not have a value then stuf will return something like

```
<ns:aoa.huisletter StUF:noValue="geenWaarde" xsi:nil="true" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"/>
```

which at this point will be transform into a dictionary

```
OrderedDict([('@noValue', 'geenWaarde'), ('@http://www.w3.org/2001/XMLSchema-instance:nil', 'true'), ('@xmlns', OrderedDict([('xsi', 'http://www.w3.org/2001/XMLSchema-instance')]))])
```

and without this check it would be returned to the SDK which would then display `[object Object]` in the input field.

Note that even numeric attributes like house number will be strings (eg. '2' and not 2) at this point so this will still work with those properly.